src/main/java/com/marklogic/appdeployer/command/SortOrderConstants.java

```diff
@@ -15,6 +15,7 @@
     public static Integer DEPLOY_MIMETYPES = 90;
 	public static Integer DEPLOY_PROTECTED_PATHS = 95;
 	public static Integer DEPLOY_QUERY_ROLESETS = 97;
+    public static Integer ASSIGN_HOSTS_TO_GROUPS = 98;
```

Comment:
Can you add a comment here to explain that this needs to be done before any databases are created, and why that is?